index.html

```diff
@@ -12,15 +12,15 @@
 	<header>
 		<div class="title-date">
 			<h1>Todo List</h1>
-			<p id="todayDate">date</p> <!--this will display the date-->
+			<time id="todayDate"></time>
 		</div>
 		<div class="input">
-			<input type="text" id="userInput" placeholder="Write down your goals!">
+			<input id="userInput" placeholder="Write down your goals!">
 			<button id="addButton">+</button>
```

Comment:
This button should be a submit button
```suggestion
			<button type="submit" id="addButton">+</button>
```